packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -440,14 +454,12 @@ contract ColosseumTest is Colosseum_Initializer {
     }
 
     function test_approveChallenge_succeeds() public {
-        test_proveFault_succeeds();
 
         uint256 outputIndex = oracle.latestOutputIndex();
         Types.CheckpointOutput memory prevOutput = oracle.getL2Output(outputIndex);
+        test_proveFault_succeeds();
 
-        vm.prank(guardian);
-        colosseum.approveChallenge(outputIndex);
```

Comment:
If you remove this line, `test_approveChallenge_succeeds` doesn't test whether `approveChallenge` is called successfully as the name implies.

Reply:
Since the approveChallenge process takes place inside test_proveFault_succeeds, I don't think we need to call approveChallenge directly afterwards. Even if we did, it would fail because the state of the challenge would have already changed.